sourcegraph/sourcegraph
# June 28th - Update main nav in dotcom

## Description
Hide some nav items on `isSourcegraphDotCom`, to match [design specification](https://www.figma.com/file/bzrwwnI9GXjG3WbkvN2rMM/Get-Started-with-Cody?node-id=340%3A5264&mode=dev)

## Ref
[Issue](https://github.com/sourcegraph/about/issues/6320)
[Gitstart Ticket](https://clients.gitstart.com/sourcegraph/1/tickets/SGM-6320)

## Demo
![Screenshot 2023-06-27 at 10 50 47 AM](https://github.com/sourcegraph/sourcegraph/assets/89894075/dbed89a1-630b-460c-ba9d-00c55b335b14)
![Screenshot 2023-06-27 at 10 50 28 AM](https://github.com/sourcegraph/sourcegraph/assets/89894075/aa213837-a9b0-481c-ad4a-d48caabcfc10)

## Test Plan
- Run sourcegraph dotcom app on this branch
- Notice the navbar only displays the nav items as indicated on the Figma design
